Library/Formula/hugo.rb

```diff
@@ -19,6 +19,21 @@ class Hugo < Formula
       url "https://github.com/spf13/hugo/commit/967d001ebe40cfe90992d953880356a495216202.diff"
       sha256 "5a991372a54ac049783926c1993d920f46ab52563ff1877e7aaddc40b5a79e22"
     end
+
+    # Install version of fsnotify needed by stable
+    go_resource "gopkg.in/fsnotify.v0" do
+      url "https://github.com/go-fsnotify/fsnotify.git",
+        :branch => "v0",
+        :revision => "ea925a0a47d225b2ca7f9932b01d2ed4f3ec74f6"
+    end
+  end
+
+  # Install version of fsnotify needed by head
+  head do
```

Comment:
put `url "https://github.com/spf13/hugo.git"` in this block and remove `head "https://github.com/spf13/hugo.git"` above